ansible/vault_consul_integration.yml

```
@@ -18,19 +29,30 @@
 - name: Consul - create dev team policy
```

- name: Consul - create admin team policy
```suggestion
- name: Consul - create dev team policy
```